tests/contrib/django/asgi.py

```diff
@@ -1,4 +1,24 @@
+import os
+
+from channels.routing import ProtocolTypeRouter
+from channels.routing import URLRouter
 from django.core.asgi import get_asgi_application
+from django.urls import path
+
 
+# Application must be run in the project root to find this settings ex. ddtrace/
+os.environ.setdefault("DJANGO_SETTINGS_MODULE", "tests.contrib.django.django_app.settings")
 
 application = get_asgi_application()
+
+application2 = ProtocolTypeRouter(
```

Comment:
maybe `channels_application` is a better name?

Reply:
Yup, made this change.